interfaces/matlab_experimental/Phases/Interface.m

```diff
@@ -0,0 +1,155 @@
+classdef Interface < handle & ThermoPhase & Kinetics
```

Comment:
Like `Solution.m`, this file should be moved into a dedicated `base` folder.

Again, there are significant simplifications in `clib`. You can load the interface using
```c
int surf = soln_newInterface("ptcombust.yaml", "Pt_surf", 0, adj); 
int gas = soln_adjacent(surf, 0);
```
with `adj` being an empty pointer. Alternatively, you can pass the gas phase in the `adj` argument.